build/rtsp-stream.yml

```diff
@@ -10,3 +10,9 @@ endpoints:
   list:
     enabled: true
     secret: macilaci
+
+listen:
+  - id: sample
```

Comment:
Yeah, I noticed that mistake in the yml file today. The sub-library does not allow setting the ID externally, which is why I setup the alias approach. That should have been "name" rather than "id".